tests/plugins/images/test_forms.py

```diff
@@ -0,0 +1,15 @@
+from django.test import TestCase
+from django.utils.translation import gettext
+
+from wiki.plugins.images.forms import PurgeForm
+
+
+class PurgeFormTest(TestCase):
```

Comment:
I see that there are two forms with more or less the same content, only difference is the label on `confirm`. See `DeleteForm` in the `attachments` plugin. Perhaps it's better to refractor this out to a single form.

Reply:
Yup, and now that there is test coverage, it would make such a change easier :)